Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.py

```diff
@@ -87,12 +87,16 @@ async def handle_post(
 
     if demisto.params().get('store_samples'):
         try:
-            sample_events_to_store.append(incident)
-            integration_context = get_integration_context()
-            sample_events = deque(json.loads(integration_context.get('sample_events', '[]')), maxlen=20)
-            sample_events += sample_events_to_store
-            integration_context['sample_events'] = list(sample_events)
-            set_to_integration_context_with_retries(integration_context)
+            incident_obj_size = sys.getsizeof(incident)
```

Comment:
I think this doesn't do a recursive calculation. Please check.

Reply:
right
implemented an `actualsize` func